code/math/vecmat.cpp

```diff
@@ -2954,6 +2954,48 @@ void vm_interpolate_angles_quick(angles *dest0, angles *src0, angles *src1, floa
 	dest0->b = src0->b + (arc_measures.b * interp_perc);
 }
 
+// Interpolate between two matrices, using t as a percentage progress between them.
+// Intended values for t are [0.0f, 1.0f], but 1.0f is not checked, as you could conceivably
```

Comment:
"1.0f is not checked" might imply that 1.0 doesn't actually work.  Instead, say that `t` is not restricted to that range.  Incidentally, negative values are also mathematically valid -- for the same reason as values >1.